api-reference/beta/toc.yml

```
@@ -319,6 +319,8 @@ items:
           href: api/user-delete.md
         - name: Change password
           href: api/user-changepassword.md
+        - name: Get authentication
+          href: api/authentication-get.md
         - name: Get delta
```

Delete authentication-get.md to resolve the API doctor error
```suggestion
          href: api/user-changepassword.md
        - name: Get delta
```